boards/posix/native_posix/main.c

```diff
@@ -120,15 +121,50 @@ int main(int argc, char *argv[])
 
 #else /* CONFIG_ARCH_POSIX_LIBFUZZER */
 
+static const uint8_t *posix_fuzz_buf;
+static size_t posix_fuzz_sz;
+static K_SEM_DEFINE(fuzz_sem, 0, K_SEM_MAX_LIMIT);
+
+static bool input_result_valid;
+static int input_result;
+
+static void fuzz_isr(const void *arg)
+{
+	/* We could call check0() to execute the fuzz case here, but
+	 * pass it through to the main thread instead to get more OS
+	 * coverage.
+	 */
+	k_sem_give(&fuzz_sem);
+}
+
+int _posix_zephyr_main(void)
+{
+	int fuzzer_test_one_input(const uint8_t *data, size_t sz);
```

Comment:
This would be an expected interface which needs to provided by any app that uses this, but there is no description of it anywhere. This should have been defined in a header and properly commented. (anyhow, this comment is mute given the main review comment)